src/component/app/RegisteredUserRoute.tsx

```diff
@@ -19,15 +19,16 @@ export function RegisteredUserRoute(props: RegisteredUserRouteProps) {
 
   if (userLoading) return <span>Loading user...</span>;
 
-  if (userError) return <Navigate to={"../signin"} />;
-
-  if (!user) {
-    return <Navigate to={"../profile"} />;
+  const returnTo = window.location.pathname + window.location.search;
+  if (userError) {
+    return <Navigate to="/signin" state={{ returnTo }} />;
```

Comment:
The user is authenticated according to auth0, but there was a database error. Not sure what to do, just redirect to signin.